netbox/circuits/views.py

```diff
@@ -156,19 +147,22 @@ class ProviderNetworkListView(generic.ObjectListView):
 
 
 @register_model_view(ProviderNetwork)
-class ProviderNetworkView(generic.ObjectView):
+class ProviderNetworkView(GetRelatedModelsMixin, generic.ObjectView):
     queryset = ProviderNetwork.objects.all()
 
     def get_extra_context(self, request, instance):
-        related_models = (
-            (
-                Circuit.objects.restrict(request.user, 'view').filter(terminations__provider_network=instance),
-                'provider_network_id',
-            ),
-        )
-
         return {
-            'related_models': related_models,
+            'related_models': self.get_related_models(
+                request,
+                instance,
+                [CircuitTermination],
```

Comment:
We should be able to include circuit terminations now that #15496 has been completed.